R/hasi.R

```diff
@@ -49,6 +49,17 @@
 hasi_r_num <- function(patientID, visitDY, BodySite, BSA, InflammColorChg, Induration, OpenSkinSurface, Tunnels) {
   checkmate::assert_character(patientID, any.missing = FALSE, null.ok = FALSE)
   checkmate::assert_character(BodySite, any.missing = FALSE, null.ok = FALSE)
+  checkmate::assert_subset(BodySite, choices = c("Right Axilla",
```

Comment:
Please change this to `assert_permutation`, otherwise, other values can be included as long as these are included.

Reply:
My apologies, this function did not seem to appear for me inside checkmate at first. (I fixed this by reinstalling checkmate)

Would you want this to give an error when there is a missing site? For example, with `assert_permutation()`, my first test no longer works, since it contains a single body site. (A single value is not asserting as a permutation of the group of expected values.)

Reply:
Yes, I'm pretty sure that it should give an error since you can't calculate HASI without all body sites.  I could also imagine a scenario where it would be a warning and it would return an NA value when a site is missing.